### ISSUE TITLE
Minor sidebar annoyance

### ISSUE THREAD

**Issue by 0skar2** [zen-browser/desktop#2898]
### Captchas

- [ ] I have read the instructions.
- [ ] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Right now the sidebar, in floating mode, exits whenever i focus the page, which is annoying (even worse, switches back to default "Welcome to Zen Sidebar" panel).

I'd like to suggest a toggle, to make floating sidebar behave the same as one in pinned mode (except... its floating, obviously). That would make floating sidebar even more useful.
nya~

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.18

### What platform are you seeing the problem on?

Linux

**Comment by universish** [comment]
I don't understand your explanation, could you please explain the problem more clearly.
What steps do I need to take to reproduce the problem? Can you explain step by step.

**Comment by 0skar2** [comment]
Its not a bug but rather an inconvenience,
just put your sidebar (not tabbar) into floating mode, everytime you click elsewhere, the sidebar exits, and switches back to, whatever webpanel i had opened, to the default one.

**Comment by universish** [comment]
How to activate floating mode?

**Comment by 0skar2** [comment]
~~are you serious~~
you should see the "minimize" button on sidebar, it turns it into floating mode
![image](https://github.com/user-attachments/assets/87f5b6d1-9dd9-4f9f-b0ca-5b34c04cec23)

**Comment by universish** [comment]
There is no such thing as “floating mode”. Please choose the right words so that we can help you.
There is compact mode. In the compact mode settings there is “Toggle Floating Sidebar”. Shortcut : Alt + Ctrl + S
 Are you talking about the “Floating Sidebar” feature?
Attention. This is not a mod.

**Comment by 0skar2** [comment]
You didnt understand me, i am talking about *this sidebar*, which HAS floating mode, and i can drag it anywhere i want
![image](https://github.com/user-attachments/assets/8a445963-0ce8-447b-a14d-50272f2d8a26)

**Comment by universish** [comment]
“Side panels”

**Comment by JSGRD22** [comment]
Cannot reproduce. Web panels stays open when focusing the page. Do you still experience this issue?

**Comment by Entrivax** [comment]
It looks like you can prevent the web panel to automatically close by unchecking the following option: **Settings** -> **Look and Feel** -> **Side Web Panels** -> ***Close the panel when it loses focus if it's not pinned***

![Image](https://github.com/user-attachments/assets/ef0973af-124d-47f0-b1ea-1bf71880417e)

Then the web panel should stay open when it loses focus

**Comment by JSGRD22** [comment]
@0skar2 can you verify if this works?

**Comment by 0skar2** [comment]
> [@0skar2](https://github.com/0skar2) can you verify if this works?

it indeed works
